src/components/Connections.tsx

```diff
@@ -28,7 +28,8 @@ export function Connections({ peers, onSelectPeer, getPeer, onRemovePeer }: Prop
         return (
             <div onClick={handlePeerClick(peer)}>
                 {/* TODO: replace with peer name */}
-                {peer.id}
+                {/* changed to peer name*/}
```

Comment:
github highlights whats changed, we dont need comments like these to notify what has been done. also, the todo comment above can also be removed now